docs/reference/aggregations/bucket/ipprefix-aggregation.asciidoc

```diff
@@ -0,0 +1,334 @@
+[[search-aggregations-bucket-ipprefix-aggregation]]
+=== IP prefix aggregation
+++++
+<titleabbrev>IP prefix</titleabbrev>
+++++
+IP addresses can be aggregated according to their network or sub-network. IP addresses consists of two groups of bits: the most significant bits which represent the network prefix, and the least significant bits which represent the host.
```

Comment:
I think the first sentence should have a stronger description of the aggregation. Something like:

`A bucket aggregation that groups documents based on the network or sub-network of an IP address.`

The second sentence has a subject-verb agreement error:
```suggestion
IP addresses can be aggregated according to their network or sub-network. An IP address consists of two groups of bits: the most significant bits which represent the network prefix, and the least significant bits which represent the host.
```

Reply:
Yes, you are right...I thought about that. I think favouring conciseness assuming uses do know what we are talking about is important.